pages/travel-and-leave/travel-and-leave-policies/first-time-travel-get-in-concur.md

```diff
@@ -69,7 +69,7 @@ GSA travel card]({% page "/first-time-travel-travel-card" %}).
 ### 2. Check that you have completed required training
 
 Ensure you have completed the mandatory security and privacy training required
-of all GSA staff, assigned to you in OLU (https://gsaolu.gsa.gov). [Follow the
+of all GSA staff, assigned to you in [OLU](https://gsaolu.gsa.gov). [Follow the
 [instructions]({% page "/training-and-development/olu/#help-with-olu" %})
```

Comment:
@nicoleslaw there is still an unescaped bracket

Reply:
Updated--sorry I missed that. Still waiting on my text editor